app/services/regenerate_stat_service.rb

```diff
@@ -0,0 +1,17 @@
+class RegenerateStatService
+  class << self
+    def anime_genre_breakdown
+      User.where(id: LibraryEntry.select(:user_id).where(media_type: 'Anime'))
```

Comment:
I added the `.where(media-type: 'Anime')` because no point running it if they only have manga library entries.